src/maker/mod.rs

```diff
@@ -35,7 +35,12 @@ use crate::{
 
 use crate::maker::error::MakerError;
 
-/// Start the Maker server loop
+/// This function initializes and starts the Maker server, handling connections and various
+/// aspects of the Maker's behavior.
+///
+/// The `start_maker_server` function is the entry point for the Maker server. It initializes and
+/// starts the server, managing connections and ensuring the Maker's functionality.
+
```

Comment:
The second line feels extra.. just the first will do for now..

Reply:
Done with this [commit](https://github.com/utxo-teleport/teleport-transactions/pull/50/commits/3e02d382711c643c5dc6f21b8f6536b251e3da1c).